hack/generate/csv.sh

```diff
@@ -153,6 +153,7 @@ for name in "${kafka_images[@]}"; do
 done
 
 # Add Knative Kafka version to the downstream operator
+add_downstream_operator_deployment_env "$target" "CURRENT_VERSION" "$(metadata.get project.version)"
```

Comment:
is `CURRENT_VERSION` a good name?

/cc @mgencur

Reply:
We can iterate on the name if we have a better name, any suggestions?